src/CharCreation/CharCreation.elm

```diff
@@ -96,7 +95,7 @@ view (A model) =
                         ]
                     , div []
                         [ Attributes.view model.attributes
-                            |> Html.App.map Attribute
+                            |> Html.map Attribute
```

Comment:
It makes more sense to me. maps the Attribute function within the Html structure. 👍